.gitignore

```diff
@@ -1,3 +1,7 @@
 .idea/
 *.iml
 target/
```

Comment:
@vshaipov good morning and thank you for the changes. 

Why these changes are required? Eclipse?